app/controllers/address/AddressController.scala

```diff
@@ -64,7 +65,7 @@ abstract class AddressController @Inject()(
 
   def internalServerError(implicit userRequest: UserRequest[_]): Result =
```

Comment:
Move this to the RenderErrors thing and it will mean you won't have to inject in ErrorViews here and will tidy things up in general